src/Mvc/Extensions.ApiDescription.Server/src/build/Microsoft.Extensions.ApiDescription.Server.targets

```diff
@@ -16,7 +16,7 @@
   </ItemGroup>
 
   <Target Name="OpenApiGetDocuments" Returns="@(_OpenApiProjectDocuments)">
-    <Error Text="OpenAPI document generation is disabled. Add '<OpenApiGenerateDocuments>true</OpenApiGenerateDocuments>' to the project."
+    <Error Text="OpenAPI document generation is disabled. Add '&lt;OpenApiGenerateDocuments>true&lt;/OpenApiGenerateDocuments>' to the project."
```

Comment:
Don't you need to do the `&gt;`s as well?

Reply:
The `>` character is fine without escaping in attribute values or text. You'll sometimes see it escaped nonetheless but that's more about balance or consistency than anything else (`msbuild` is more stringent than some XML parsers but not extreme.)

I've tested with `dotnet build`, `dotnet msbuild` and `msbuild` and none have issues w/ the updated file.

Reply:
Personally I would escape the `>` to `&gt;` just for consistency, but it's nothing I'd hold the PR over as long as things pass.